spyder/plugins/profiler/widgets/main_widget.py

```diff
@@ -738,7 +750,8 @@ def hide_diff_cols(self, hide):
 
     def save_data(self, filename):
         """Save profiler data."""
-        self.stats1[0].dump_stats(filename)
+        if len(self.stats1) >0:
```

Comment:
Missing space after operator
```suggestion
        if len(self.stats1) > 0:
```